protocols.csv

```diff
@@ -23,9 +23,10 @@ code,	size,	name,	comment
 448,	0,	tls,	Transport Layer Security
 449,	V,	sni,	Server Name Indication RFC 6066 § 3
 454,	0,	noise,
-460,	0,	quic,
+460,  0,	quic-draft29,
```

Comment:
Let's not make this rename change yet, because I think it may break parsing existing strings.

Reply:
Reverted this change.